### PR TITLE
style: fix lint issues

### PR DESCRIPTION
**Pull request by hkang1**
## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/types.ts [hunk 1]
```diff
@@ -199,16 +199,16 @@ export type RecentEvent = {
...
   };
-}
+};
 
-export type AnyTask = CommandTask | ExperimentTask
+export type AnyTask = CommandTask | ExperimentTask;
 export type RecentTask = AnyTask & RecentEvent;
-export type RecentCommandTask = CommandTask & RecentEvent
-export type RecentExperimentTask = ExperimentTask & RecentEvent
+export type RecentCommandTask = CommandTask & RecentEvent;
```
Comment (hamidzr) [inline webui/react/src/types.ts:206]: should we make these Error level failures so they break CI test?

Comment (hamidzr) [reply webui/react/src/types.ts:206]: + the multiple space rule.

Comment (hkang1) [reply webui/react/src/types.ts:206]: yep I like that. I wasn't sure if that was too severe but with auto fix I think that's reasonable.

Comment (hkang1) [reply webui/react/src/types.ts:206]: I'll update this PR to make it error level.

**Pull request merged**